easybuild/easyconfigs/o/openjpeg/openjpeg-2.3.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,48 @@
+#
+# this file is for building the openjpeg library in Easybuild
+# Author: Fenglai Liu  (fenglai.liu@vanderbilt.edu)
+# Vanderbilt University
+#
+
+easyblock = 'CMakeMake'
+
+name = 'openjpeg'
+version = '2.3.0'
+
+homepage = 'http://www.openjpeg.org/'
+
+description = """
+OpenJPEG is an open-source JPEG 2000 codec written in C language. It has been
+developed in order to promote the use of JPEG 2000, a still-image compression
+standard from the Joint Photographic Experts Group.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [
+    'https://github.com/uclouvain/openjpeg/archive'
+]
+sources = ['v%(version)s.tar.gz']
+checksums = ['3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a']
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('CMake',    '3.9.5')
+]
+
+# for running the binary of openjpeg you also need the libraries like zlib etc.
+dependencies = [
+    ('zlib',     '1.2.11'),
+    ('libpng',   '1.6.32'),
+    ('LibTIFF',  '4.0.9')
+]
+
+# enable the optimizations
+configopts = '-DCMAKE_BUILD_TYPE=Release '
+
+sanity_check_paths = {
+    'files': ['bin/opj_decompress', 'include/openjpeg-2.3/openjpeg.h', 'lib/libopenjp2.so'],
+    'dirs': []
+}
+
+moduleclass = 'vis'
```

Comment:
@murfreesboro `lib` is probably more appropriate here?

Reply:
This is not a pure library. They also provide binary to compress/decompress pictures. So I believe it's better to go for "vis" group.